src/client/components/SelectionPage/SelectionPage.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import "./Selectionpage.css";
+import Radio from '../Radio/ButtonRadio'
+import HYFLogo from './HYF.logo.svg';
+
+const SelectOption=()=>{
```

Comment:
How did you manage to push this, formatting is way off in this file.

Reply:
i took it from README.md because i couldn't find that logo in other format